common/scoring.ts

```diff
@@ -62,7 +63,18 @@ export function scoreUsersByContract(contract: Contract, bets: Bet[]) {
       return { userId, payout }
     })
 
-  const netPayouts = [...resolvePayouts, ...salePayouts, ...investments]
+  const investmentValues = openBets.map((bet) => {
```

Comment:
investmentValues calculates the investmentValue for each open bet in the contract (only if the contract is not yet resolved).